modules/core/src/test/java/org/apache/ignite/internal/util/DisabledStripedExecutorTest.java

```diff
@@ -0,0 +1,113 @@
+/*
```

Comment:
You need to add more tests including full api and fault-tolerance tests with disabled striped pool.

Reply:
Added DisabledStripedExecutorFailoverTest and DisabledStripedExecutorFullApiTest. Please let me know if we need more tests.